### PR TITLE
Document differences between dasbus and pydbus

### PR DESCRIPTION
**Pull request by poncovka**
Provide a list of changes in API and the implementation.

### CODE REVIEW

File: docs/pydbus.rst [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  to watch a DBus name.
+
+* Acquiring DBus names: Call the :meth:`register_service <dasbus.connection.MessageBus.register_service>`
+  method to acquire a DBus name.
+
+* Providing XML specifications: Use the ``__dbus_xml__`` attribute to provide the XML
+  specification of a DBus object. Or you can generate it from the code using the
+  :func:`dbus_interface <dasbus.server.interface.dbus_interface>` decorator.
+
+* No support for polkit
```
Comment (jkonecny12) [inline docs/pydbus.rst:76]:
```suggestion
* No support for polkit.
```

File: docs/pydbus.rst [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  the GLib event loop. However, the GLib part of dasbus is separated from the rest of the code,
+  so it shouldn't be too difficult to add support for a different backend. It would be necessary
+  to replace :class:`dasbus.typing.Variant` and :class:`dasbus.typing.VariantType` with their
+  abstractions and reorganize the code.
+
+* No support for org.freedesktop.DBus.ObjectManager: There is no support for object managers,
+  however the :class:`DBus containers <dasbus.server.container.DBusContainer>` could be a good
+  starting point.
+
+* No support for unix file descriptors
```
Comment (jkonecny12) [inline docs/pydbus.rst:91]:
```suggestion
* No support for unix file descriptors.
```

File: docs/pydbus.rst (hunk 1 above)
Comment (jkonecny12) [reply docs/pydbus.rst:76]: Also I wonder why do you have polkit here? What is the reason to have polkit support? System services?

File: docs/pydbus.rst (hunk 2 above)
Comment (jkonecny12) [reply docs/pydbus.rst:91]: Similar question here? Do you mean sending FDs by DBus and understanding the type? If yes, then it should be easy to use just `int` to send it.

File: docs/pydbus.rst (hunk 1 above)
Comment (poncovka) [reply docs/pydbus.rst:76]: Pydbus has support for polkit for some reason, so it should be documented that dasbus does not.

File: docs/pydbus.rst (hunk 2 above)
Comment (poncovka) [reply docs/pydbus.rst:91]: DBus has a special data type for file descriptors and it is necessary to call some special GLib functions. There is an open issue for pydbus, so it is fair to mention it here.

File: docs/pydbus.rst (hunk 1 above)
Comment (poncovka) [reply docs/pydbus.rst:76]: Btw. it is not a sentence, so I don't think that it should end with a dot.

File: docs/pydbus.rst (hunk 2 above)
Comment (poncovka) [reply docs/pydbus.rst:91]: Same here.

File: docs/pydbus.rst (hunk 1 above)
Comment (VladimirSlavik) [reply docs/pydbus.rst:76]: My inner writer wants to complain that you must have all items of a bullet list be sentences or not consistently. Thus the argument would be invalid. But you don't want to listen to that voice too much. It won't settle for anything less than _perfection_ :-/

Comment (poncovka) [reply docs/pydbus.rst]: Ok, I have changed this item.

File: docs/pydbus.rst (hunk 2 above)
Comment (poncovka) [reply docs/pydbus.rst]: This item has changed as well.

**Pull request merged**